<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label0">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爬取进度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button0">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text0">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爬取进度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爬取进度</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爬取进度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doCrawl_3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爬取进度</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stopDoCrawl_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爬取进度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doCrawl_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爬取进度</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>500</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爬取进度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button8">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>爬取进度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button6">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
